includes/wp-admin/class-settings-fields.php

```diff
@@ -80,14 +80,16 @@ public static function register_settings_fields() {
 			);
 		}
 
-		add_settings_field(
-			'activitypub_max_image_attachments',
-			__( 'Media attachments', 'activitypub' ),
-			array( self::class, 'render_max_image_attachments_field' ),
-			'activitypub_settings',
-			'activitypub_activities',
-			array( 'label_for' => 'activitypub_max_image_attachments' )
-		);
+		if ( \is_plugin_active( 'classic-editor/classic-editor.php' ) ) {
```

Comment:
can we reuse the `site_supports_blocks` function and add the plugin check there?